app/api/search/search.js

```diff
@@ -557,13 +557,15 @@ async function searchTypeFromSearchTermValidity(searchTerm) {
 const buildQuery = async (query, language, user, resources, includeReviewAggregations) => {
   const [templates, dictionaries] = resources;
   const textFieldsToSearch = _getTextFields(query, templates);
-  const searchTextType = query.searchTerm
-    ? await searchTypeFromSearchTermValidity(query.searchTerm)
+  // GitHub: Ability to use curly and straight quotes interchangeably #4963
```

Comment:
As part of our coding style, we don't introduce comments in the code unless absolutely necessary. This particular piece of information is provided by the git history. The commit messages usually hold this information and GitHub will add the PR number to the squashed commit during the merge. From there anyone can find the related issue on GitHub. Let's remove it :)

Reply:
I'll remove it! Thank you